### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by SebastianM-C**
This PR add a short description of what is needed in order to use RegistryCI with a custom registry.

The information was mostly provided by @ericphanson, @christopher-dG and @DilumAluthge on slack.

### CODE REVIEW

**Comment by SebastianM-C** [comment]
Regarding the `AutoMerge` part, I'm not sure if that is all that is required as I have not yet tested that in my custom registry.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,80 @@ For the list of automatic merging guidelines, please see the [General registry R
...
+    merge_new_packages = ENV["MERGE_NEW_PACKAGES"] == "true",
+    merge_new_versions = ENV["MERGE_NEW_VERSIONS"] == "true",
+    new_package_waiting_period = Day(3),
+    new_jll_package_waiting_period = Minute(20),
+    new_version_waiting_period = Minute(10),
+    new_jll_version_waiting_period = Minute(10),
+    registry = "JuliaRegistries/General",
+    tagbot_enabled = true,
+    authorized_authors = String["JuliaRegistrator"],
+    authorized_authors_special_jll_exceptions = String["jlbuild"],
```
[the remark this answers is not in the document]
Comment (ericphanson) [reply README.md:91]: In General, the special user `jlbuild` is used to register JLL packages and gets to skip the 3 day waiting period. Probably most custom registries don't want that.

**Review by ericphanson — COMMENTED** [review]
This is great, thanks!

I think it might be good to adjust the automerge script directly too.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SebastianM-C) [reply README.md:91]: Thanks, I wasn't sure about that.

**Review by ericphanson — APPROVED** [review]

**Comment by ericphanson** [comment]
bors merge

**Comment by DilumAluthge** [comment]
Needs a version bump.

**Comment by ericphanson** [comment]
maybe https://github.com/JuliaRegistries/RegistryCI.jl/pull/372 will fix that

**Comment by SebastianM-C** [comment]
@DilumAluthge done

**Comment by ericphanson** [comment]
bors merge

**Comment by DilumAluthge** [comment]
Ah, I just landed #373. Bump it again.

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "RegistryCI"
 uuid = "0c95cc5f-2f7e-43fe-82dd-79dbcba86b32"
 authors = ["Dilum Aluthge <dilum@aluthge.com>", "Fredrik Ekre <ekrefredrik@gmail.com>", "contributors"]
-version = "6.8.7"
+version = "6.8.8"
```
Comment (ericphanson) [inline Project.toml:4]:
```suggestion
version = "6.8.9”
```

**Comment by ericphanson** [comment]
bors merge

**Comment by DilumAluthge** [comment]
GitHub Actions is currently experiencing degraded performance: https://githubstatus.com

We'll probably need to wait until that is fixed.

**Comment by DilumAluthge** [comment]
bors merge

**Comment by DilumAluthge** [comment]
bors merge-

**Comment by DilumAluthge** [comment]
The problem was the quotation marks.

**Comment by DilumAluthge** [comment]
bors merge

**Pull request merged**